packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -608,3 +608,9 @@ export class PreferencesTreeWidget extends TreeWidget {
         return a.localeCompare(b, undefined, { ignorePunctuation: true });
     }
 }
+
+export const PREFERENCE_IDS = {
```

Comment:
Why do we need the new `const`?
It doesn't look to be used anywhere at the moment.

Reply:
The thought behind it was to facilitate the rebinding of the different preferences widgets, as I could not find a way to successfully replace one without assigning its ID to a new widget.

If this is not the proper way to do it, I would love a nudge in the right direction. I will be happy to remove the 'const' if needed.

Reply:
I'm not sure it's really useful or needed as you can simply use the `id` directly (the `ids` are static):
Example:
```ts
 PreferencesTreeWidget.ID,
```

Reply:
That's a good point! I'll push up the change after a quick test run.